prov/gni/src/gnix_rma.c

```diff
@@ -79,17 +114,20 @@ static int __gnix_rma_send_completion(struct gnix_fid_ep *ep,
 				  "_gnix_cntr_inc() failed: %d\n", rc);
 	}
 
-	return rc;
+	return FI_SUCCESS;
```

Comment:
Is it documented somewhere that failed counter increment only is a warning?  If not, I think we should put it into our man page under CAVEATS.